### DISCUSSION TITLE
"Save & Add Another" Button in New record - Form

### DISCUSSION THREAD

**Discussion by muksidin** [nocodb/nocodb#7793] [Features]
Hi All, how about add "Save & Add Another" button beside "Save" Button. So when i use "Save & Add Another" the form is resetted to empty form. I think this feature is usefull and i need this. Thanks
![Screenshot_1](https://github.com/nocodb/nocodb/assets/5665115/ddb0c21e-421b-49b2-9ad6-5cf5fd112f8f)

**Comment by dstala** [comment]
I would recommend you to use form view for data entry & then enable show `Submit another form` button in the form view. 
Note that, form view in the builder can also be used for data entry without sharing a public link.
